app/javascript/stylesheets/components/_stats.scss

```
@@ -38,3 +38,44 @@ ul.rdv-stats-legend {
   justify-content: center;
   align-items: center;
 }
+
+.rdv-stats-hero-figure {
+  text-align: center;
+  color: var(--text-action-high-blue-france);
+
+  p:first-child {
+    font-size: 64px;
+    line-height: 72px;
+    font-weight: 800;
+    @include media-breakpoint-up(md) {
```

je dis peut-être n'importe quoi, et c'est peut-être les même valeurs, mais est-ce que avec cette variable on peut utiliser le breakpoint défini par le DSFR plutôt que celui de bootstrap ?
```suggestion
    @include media-breakpoint-up($bp-md) {
```

---

c’est une bonne suggestion ! j’ai un peu creusé : 
- `bp-md` n’apparaît nulle part dans tout `node_modules/@gouvfr/dsfr/ist` 🤷 leur doc est vraiment mystérieuse sur les tokens
- dans le `dist/dsfr.main.css` les breakpoints sont hardcodés dans les media queries
- ce `media-breakpoint-up(md)` est compilé en `@media (min-width: 768px) {`
- dans https://www.systeme-de-design.gouv.fr/version-courante/fr/fondamentaux/grille-et-points-de-rupture on voit que c’est le même breakpoint md = 768px

on s’en sort donc bien 😅